src/libs/conduit/python/conduit_python.cpp

```diff
@@ -271,7 +315,16 @@ PyConduit_DataType_new(PyTypeObject* type,
         return (NULL);
     }
 
+    #ifdef Py_LIMITED_API
+    allocfunc tp_alloc = (allocfunc)PyType_GetSlot(type, Py_tp_alloc);
+    if(!tp_alloc)
+    {
+      return NULL;
+    }
+    PyConduit_DataType *self = (PyConduit_DataType*)tp_alloc(type, 0);
+    #else
     PyConduit_DataType *self = (PyConduit_DataType*)type->tp_alloc(type, 0);
+    #endif
```

Comment:
You can probably just use [`PyType_GenericAlloc`](https://docs.python.org/3/c-api/type.html#c.PyType_GenericAlloc) instead.

Reply:
Thanks I wasn't aware of this function .